src/sql/parts/notebook/notebook.component.ts

```diff
@@ -159,12 +171,23 @@ export class NotebookComponent extends AngularDisposable implements OnInit {
 		await model.requestModelLoad(this.notebookParams.isTrusted);
 		model.contentChanged((change) => this.handleContentChanged(change));
 		this._model = model;
+		this.updateToolbarComponents(this._model.trustedMode);
 		this._register(model);
 		this._modelRegisteredDeferred.resolve(this._model);
 		model.backgroundStartSession();
 		this._changeRef.detectChanges();
 	}
 
+	//Updates toolbar components
```

Comment:
Nitpick: we normally have a space after `//`, so `// Updates toolbar components`